src/main/java/com/example/petbuddybackend/entity/user/Caretaker.java

```diff
@@ -60,6 +61,9 @@ public class Caretaker {
     @Builder.Default
     private Set<CaretakerNotification> notifications = new HashSet<>();
 
+    @OneToMany(mappedBy = "caretaker", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @Builder.Default
+    private List<Care> cares = new ArrayList<>();
```

Comment:
Tu też się zastanawiam czy to `CascadeType.ALL` byłoby ok. Myślę że opieki to mogły by być dane historyczne i nawet jakby usunąć/zdezaktywować konto opiekuna to można je przechowywać